app/jobs/calendar_importer/parsers/manchester_uni.rb

```diff
@@ -4,6 +4,9 @@
 
 module CalendarImporter::Parsers
   class ManchesterUni < Xml
+    NAME = 'Manchester University'
```

Comment:
We should prob have this hidden from the list of generic importers as its a bit confusing. Maybe we could have a "show to public" bool for the importer that prevents it showing up? Or do the move to a subfolder thing I suggested before?

Reply:
I think my pref would prob be the latter just to make it ultra clear whats what in the codebase